app/scripts/migrations/156.ts

```diff
@@ -39,7 +39,8 @@ function transformState(state: Record<string, unknown>) {
   const newState = cloneDeep(state);
 
   if (!hasProperty(newState, 'NftController')) {
-    throw new Error(`newState.NftController must be present`);
+    console.error(`newState.NftController must be present`);
```

Comment:
Nit: I know I didn't differentiate in our earlier discussion, but maybe `console.warn` is more appropriate here since this isn't really an error case?

Not a big deal though, it solves the bug either way

```suggestion
    console.warn(`newState.NftController must be present`);
```